db/migrate/20170529215453_create_organizations.rb

```diff
@@ -0,0 +1,9 @@
+class CreateOrganizations < ActiveRecord::Migration
+  def change
+    create_table :organizations do |t|
+      t.string :name
```

Comment:
Since `name` is a required field, it also makes sense to put it here:
`t.string :name, null: false`